### ISSUE TITLE
Add official position on example ranges to style.md

### ISSUE THREAD

**Issue by chbrown** [semprag/semprag.github.com#7]
What to do with example ranges? Options:

| Format | Comment |
|:-------|:--------|
| (1)--(2)        | n-dash conflicts with `\dash` ultimatum
| (1) -- (2)      | precedent of page range does not have surrounding space
| (1)-(2)         |
| (1-2)           | might conflict with systems that might label a single example as (1-2) or with built-in `\ref`
| (1) to (2)      | verbose
| (1) through (2) | verbose

Corollary: what about sub-examples?

| Format | Comment |
|:-------|:--------|
| (1a)--(1c) |
| (1a--c)    | might not play nice with built-in `\ref`
| (1a-c)     | ditto
| (1b)--(4b) | not unheard of!

**Comment by fintelkai** [comment]
Definitely, (1)--(2), on the model of page ranges and other numerical ranges. The `\dash` edict is about anything that should be an em-dash: parenthetical remarks.

**Comment by fintelkai** [comment]
I'm vacillating between (1a--c) and (1a-c). And when the main number is different, definitely (1b)--(4b).
